### PR TITLE
Add `add_vmanage`, `edit_vmanage` and `get_connected_devices` endpoints for cluster management

### PR DESCRIPTION
**Pull request by przsus**
# Pull Request summary:
Add `add_vmanage`, `edit_vmanage` and `get_connected_devices` endpoints for cluster management.
Add Persona enum.

# Checklist:
- [x] Make sure to run pre-commit before committing changes
- [x] Make sure all checks have passed
- [x] PR description is clear and comprehensive
- [x] Mentioned the issue that this PR solves (if applicable)
- [x] Make sure you test the changes

### CODE REVIEW

File: catalystwan/utils/persona.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Copyright 2024 Cisco Systems, Inc. and its affiliates
+
+from enum import Enum
+
+
+class Persona(str, Enum):
```
Comment (cicharka) [inline catalystwan/utils/persona.py:6]: Literal > Enum

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "catalystwan"
-version = "0.35.0dev2"
+version = "0.35.0dev3"
```
Comment (sbasan) [inline pyproject.toml:3]: please set: `export catalystwan_export_endpoints=1` when bumping version
so `pre-commit` will re-generate `ENDPOINTS.md`

File: catalystwan/endpoints/cluster_management.py [hunk 3]
```diff
@@ -24,12 +26,30 @@ class VManageDetails(BaseModel):
...
+class ConnectedDevice(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+    uuid: UUID
+    device_id: str = Field(serialization_alias="deviceId", validation_alias="deviceId")
+
+
+class VManageSetup(BaseModel):
+    model_config = ConfigDict(populate_by_name=True)
+    vmanage_id: Optional[str] = Field(default=None, serialization_alias="vmanageID", validation_alias="vmanageID")
+    device_ip: str = Field(default=None, serialization_alias="deviceIP", validation_alias="deviceIP")
```
Comment (sbasan) [inline catalystwan/endpoints/cluster_management.py:38]: I guess these needs to be `Optional[str]` if we allow `None` as default

Comment (przsus) [reply catalystwan/endpoints/cluster_management.py]: actually vars not marked as optional should not have default None in this case, my bad

File: pyproject.toml (hunk 2 above)
Comment (przsus) [reply pyproject.toml]: removed the bumping, since it's these changes are not that urgent to merge

**Pull request merged**